src/mugs.js

```diff
@@ -930,6 +931,9 @@ define([
             Group.init(mug, form);
             mug.p.appearance = 'field-list';
         },
+        spec: {
+            requiredAttr: { presence: "notallowed" },
+        },
```

Comment:
This shouldn't be necessary since FieldList extends Group. Ditto for Repeat below.